build.sbt

```diff
@@ -10,24 +10,49 @@ val commonSettings = androidBuildAar ++ bintrayPublishSettings ++ Seq(
   licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
 
   scalaVersion := "2.11.7",
-  crossScalaVersions := Seq("2.10.5", "2.11.7"),
+  crossScalaVersions := Seq("2.10.6", "2.11.7"),
   javacOptions ++= Seq("-source", "1.7", "-target", "1.7"),
-  scalacOptions ++= Seq("-feature", "-deprecation", "-target:jvm-1.7"),
+  scalacOptions ++= Seq(
+    "-feature",
+    "-deprecation",
+    "-target:jvm-1.7",
+    "-encoding", "UTF-8",
+    "-unchecked",
+    "-Xlint",
+    "-Yno-adapted-args",
+    "-Ywarn-dead-code",
+    "-Ywarn-numeric-widen"
+  ) ++ (scalaBinaryVersion.value match {
+    case "2.10" => Seq.empty
```

Comment:
Could you use the same technique as below in the file? Or change the other one, just so that they are uniform.
```scala
libraryDependencies ++= (CrossVersion.partialVersion(scalaVersion.value) match {
  case Some((2, 10)) ⇒
    Seq("org.scalamacros" %% "quasiquotes" % paradiseVersion)
  case _ ⇒
    Seq()
})
```